### PR TITLE
Basic blog functionality and working nav bar links

### PR DESCRIPTION
**Pull request by RosieCampbell**
It has been a *long time* since I've done web development and an even longer time since I contributed to another person's project, so I hope this is ok!

### CODE REVIEW

File: posts/posts.11tydata.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    layout: "default",
+    tags: ["posts"],
+    eleventyComputed: {
+        
+        title: data => {
+            if(data.title){
+                return data.title;
+            }
+            let str = titleCase(data.page.fileSlug);
+            str = str.replace(/-/g, ' ');
```
Comment (b3u) [inline posts/posts.11tydata.js:14]: Should these be reversed? Before you add in spaces, the `str` variable is a single word. Only the first letter is capitalized.

Comment (RosieCampbell) [reply posts/posts.11tydata.js:14]: Yes you're right - sorry! Will fix

**Comment by b3u** [comment]
This is great; thanks for the PR.
I noticed some code that we could improve, so I left a comment (see https://github.com/b3u/eleventy-garden/pull/13#discussion_r460158873). Also, I'd like to discuss #10 more, before merging this.

File: posts/posts.11tydata.js (hunk 1 above)
Comment (b3u) [reply posts/posts.11tydata.js]: Netlify is complaining. Try this:
```js
let str = data.page.fileSlug.replace(/-/g, ' ');
str =  titleCase(str);
```

**Comment by RosieCampbell** [comment]
@b3u ok cool think I've made the fix! Let me know what you're thinking re: #10

**Comment by b3u** [comment]
Since https://github.com/b3u/eleventy-garden/issues/10#issuecomment-663925453, we should probably remove code which enables the blog section (for now).

**Pull request closed without merging**